test/list_test.dart

```diff
@@ -699,4 +699,15 @@ Future<void> main([List<String>? args]) async {
     expect(team.players, isNot(players));
     expect(team.players, unorderedMatches(players));
   });
+
+  test('ManagedRealmList.length= throws', () {
```

Comment:
should we have a test for length decrease as well. Now the `removeRange` path is not tested